cryptol-remote-api/python/pyproject.toml

```diff
@@ -14,10 +14,11 @@ include = [
 
 [tool.poetry.dependencies]
 python = ">=3.8.0,<4"
-requests = "^2.31.0"
+requests = "<2.31.0.20240310"
```

Comment:
I think this constraint needs to apply to [`types-requests`](https://pypi.org/project/types-requests/) as well, since this constraint on `requests` alone doesn't appear to do the trick. (It's unclear to me what the relationship between the two packages' versions are, but better to constraint both, I'd say.)